pkg/commands/config.go

```diff
@@ -55,6 +58,7 @@ var (
 // getBaseImage returns a function that determines the base image for a given import path.
 // If the `bo.BaseImage` parameter is non-empty, it overrides base image configuration from `.ko.yaml`.
 func getBaseImage(platform string, bo *options.BuildOptions) build.GetBase {
+	v := &verifier{entries: make(map[name.Digest]*entry, 10)}
```

Comment:
Why `10` here?

Reply:
Seemed like a small enough number to be reasonable, yet avoid resizing.  I don't particularly care.

Reply:
Same. I don't think there's any reason to care about the initial size of the map, so let's just YOLO-`make` it, or `map[name.Digest]*entry{}`.